featurebyte/query_graph/node/input.py

```diff
@@ -420,10 +420,10 @@ def _derive_node_operation_info(
                 SourceDataColumn(
                     name=column.name,
                     table_id=self.parameters.id,
-                    table_type=self.parameters.type,
+                    table_type=TableDataType(self.parameters.type),
                     node_names={self.name},
                     node_name=self.name,
-                    dtype=column.dtype,
+                    dtype=DBVarType(column.dtype),
                     filter=False,
                 )
```

Comment:
Since `SourceDataColumn` is dataclass but not pydantic model, it will stores the `str` rather than enum. Without setting them to enum, it will throw many pydantic warning at https://github.com/featurebyte/featurebyte-app/blob/5b688cd5a285b5685912b739e68e2d97bf54a2dc/worker/task/feature_ideation/ideated_feature_create.py#L447.

Reply:
Below is one of the warning examples:
```
*** UserWarning: Pydantic serializer warnings:
  Expected `enum` but got `str` with value `'VARCHAR'` - serialized value may not be as expected
  Expected `enum` but got `str` with value `'INT'` - serialized value may not be as expected
  Expected `enum` but got `str` with value `'TIMESTAMP_TZ'` - serialized value may not be as expected
```